src/server/letter-lists.js

```diff
@@ -0,0 +1,42 @@
+const consonants = {
```

Comment:
As as long as node handles it i'm fine. I think we're on 6.something so should be good http://node.green/